docker-compose.yml

```diff
@@ -1,37 +1,79 @@
 # Docker Compose file for FreeCodeCamp
 #
+# if using Docker Toolbox, replace `localhost` to output form `docker-machine ip`
+#
 # Bootstrap with:
-#   docker-compose run --rm freecodecamp npm install
-#   docker-compose run --rm freecodecamp npm run only-once
+#   docker-compose run --rm freecodecamp bash
+#     npm install --unsafe-perm
+#     npm run seed
+#     exit
```

Comment:
Its difficult to copy these commands from a file like formatted like this. Can you possibly update the instructions in the docs? Or add it here in the review comment, and I can look into adding it to docs.

Reply:
It's formatted like this because of each command exec independently. Firstly, you run bash in a container. Secondly, in this bash, you run other commands. Update comment in the file.

I cannot find where instructions for using docker in the docs.

Reply:
Could we script this?